companion/lib/Internal/BuildingBlocks.ts

```diff
@@ -99,6 +99,18 @@ export class InternalBuildingBlocks implements InternalModuleFragment {
 						min: 0,
 						max: 3600 * 1000,
 					},
+					{
+						type: 'dropdown',
+						label: 'Execution mode',
+						id: 'execution_mode',
+						default: 'parallel',
+						choices: [
+							{ id: 'parallel', label: 'Parallel' },
+							{ id: 'sequential', label: 'Sequential' },
+						],
+						tooltip:
+							'Using "Sequential" will run the actions one after the other, waiting for each to complete before starting the next. This doesn\'t work for all modules.',
```

Comment:
Instead of "This doesn't work for all modules" I suggest something like: "If the module doesn't support it for a particular action, the following action will start immediately."